src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -188,13 +177,12 @@ object WorkflowDescriptor {
     }
   }
 
-  private def validateBackendOptions(id: WorkflowId, options: WorkflowOptions, backend: Backend): ErrorOr[WorkflowOptions] = {
+  private def validateBackendOptions(id: WorkflowId, options: WorkflowOptions): ErrorOr[WorkflowOptions] = {
     try {
-      backend.assertWorkflowOptions(options)
       options.successNel
     } catch {
       case e: Exception =>
-        s"Workflow $id has invalid options for backend ${backend.backendType}: ${e.getMessage}".failureNel
+        s"Workflow $id has invalid options: ${e.getMessage}".failureNel
```

Comment:
Done for the first part.
About the runtime attribute validation, if we want to put it here that would mean creating the Backend instance here itself, which again brings us back to how things were here originally (i.e. having it's reference here). We can add a generic validation here for common things like Disk, CPU, Memory etc., but for most parts, most of the information can only be comprehended by the specific backend, which currently is only known at the CallActor level. This can be up for discussion IMHO, about actually how fast we want the fail-fast behavior to act.